packages/core/src/codewhispererChat/tools/executeBash.ts

```diff
@@ -75,6 +75,7 @@ export class ExecuteBash {
             const stderrBuffer: string[] = []
 
             let firstChunk = true
+            let firstStderrChunk = true
```

Comment:
what is this flag used for?

Reply:
If it's the first chunk we add this mark down string ('```console\n') to format the code block appropriately